source/index.js

```diff
@@ -209,4 +209,8 @@ export {
 	stderrColor as supportsColorStderr,
 };
 
+export const modifiers = Object.keys(ansiStyles.modifier);
+export const foregroundColors = Object.keys(ansiStyles.color);
+export const backgroundColors = Object.keys(ansiStyles.bgColor);
```

Comment:
This makes our Chalk slower. Is this acceptable?

Reply:
That's very strange. Can you show how?

Reply:
This makes the code slightly slower, but it might be negligible.